ScanCommand/src/main/java/io/ballerina/scan/utils/ScanUtils.java

```diff
@@ -120,6 +147,129 @@ private static Target getTargetPath(Project project, String directoryName) {
         }
     }
 
+    /**
+     * Returns the {@link Path} of the html analysis report where generated issues are saved.
+     *
+     * @param issues        generated issues
+     * @param project       Ballerina project
+     * @param directoryName target directory name
+     * @return path of the html analysis report where generated issues are saved
+     */
+    public static Path generateScanReport(List<Issue> issues, Project project, String directoryName) {
+        JsonObject jsonScannedProject = new JsonObject();
+        jsonScannedProject.addProperty(SCAN_REPORT_PROJECT_NAME, project.currentPackage().packageName().toString());
+        Map<String, JsonObject> jsonScanReportPathAndFile = new HashMap<>();
```

Comment:
Including json in variable names may be redundant. Will help shorten the names too.